libs/ballot-interpreter-nh/src/interpret.rs

```diff
@@ -363,50 +406,34 @@ pub fn interpret_ballot_card(
                 &geometry,
                 &side_b.image,
                 side_b.border_inset,
+                Some(&options.election),
                 &mut side_b_debug,
             )
         },
     );
 
-    let (side_a_grid, side_a_normalized_img) = side_a_result?;
-    let (side_b_grid, side_b_normalized_img) = side_b_result?;
+    let side_a_is_front = is_side_a_the_front_side(&side_a_result, &side_b_result)?;
+    let (side_a_grid, side_a_normalized_image) = side_a_result?;
+    let (side_b_grid, side_b_normalized_image) = side_b_result?;
```

Comment:
TODO @jonahkagan: A little more work is needed here to handle the case where we only successfully read one QR code. Currently this will still early return with an error.

It might be simpler to just have a step where we correct the QR code metadata for the two sides before determining front/back.